### DISCUSSION TITLE
default.latex: \setcounter{secnumdepth}{-\maxdimen}

### DISCUSSION THREAD

**Discussion by iwelch** [jgm/pandoc#9756] [Q&A]
I thought secnumdepth should be a number from 0 to 6 (or so).  I am wondering why it is set to `-\maxnumdepth` instead of 0 to suppressed section numbers.  anyone know?

**Comment by jgm** [comment]
```
    Notes on secnumdepth:
    
    1 = Number `\section`
    0 = Number `\chapter`
    -1 = Number `\part`
    -2 = No numbering
```

**Comment by iwelch** [comment]
then why does line 348 of default.latex in https://github.com/jgm/pandoc-templates/blob/master/default.latex read:

```
\setcounter{secnumdepth}{-\maxdimen} % remove section numbering
```

minor buglet or intentional?

**Reply by jgm** [reply]
See #5351

**Comment by iwelch** [comment]
Ahhh... Koma script special thingy...
